tests/components/wallbox/test_init.py

```diff
@@ -70,6 +70,34 @@ async def test_wallbox_refresh_failed_invalid_auth(
     assert entry.state == ConfigEntryState.NOT_LOADED
 
 
+async def test_wallbox_refresh_failed_connection_error_auth(
```

Comment:
Is this test relevant to the change?

Reply:
I noticed that the codecoverage wasn't 100Pct, therefore I added this test. Honestly, it probably wasn't 100pct before the change, so we can skip it for this PR if necessary.

Reply:
Can we move this test into it's own PR as it's not related?

Reply:
@gjohansson-ST @joostlek I removed the test, will make a seperate PR for it.